### PR TITLE
Small improvements: indentation and syntax

### PR DESCRIPTION
**Pull request by nikiforosbotis**
Minor improvements and additions in the indentation (1 line instead of 2) and in the syntax of some comments.

### CODE REVIEW

File: core/src/main/java/org/elasticsearch/index/store/Store.java [hunk 1]
```diff
@@ -641,9 +639,9 @@ final void verifyAfterCleanup(MetadataSnapshot sourceMetaData, MetadataSnapshot
                 for (StoreFileMetaData meta : recoveryDiff.different) {
                     StoreFileMetaData local = targetMetaData.get(meta.name());
                     StoreFileMetaData remote = sourceMetaData.get(meta.name());
-                    // if we have different files the they must have no checksums otherwise something went wrong during recovery.
-                    // we have that problem when we have an empty index is only a segments_1 file then we can't tell if it's a Lucene 4.8 file
-                    // and therefore no checksum. That isn't much of a problem since we simply copy it over anyway but those files come out as
+                    // if we have different files then they must have no checksums; otherwise something went wrong during recovery.
+                    // we have that problem when we have an empty index (is only a segments_1 file) then we can't tell if it's a Lucene 4.8 file
```
Comment (nik9000) [inline core/src/main/java/org/elasticsearch/index/store/Store.java]: I think this should be: "we have an empty index is only a segments_1 file *so* we can't tell"

File: core/src/main/java/org/elasticsearch/index/store/Store.java [hunk 2]
```diff
@@ -705,7 +702,7 @@ public String toString() {
     /**
      * Represents a snapshot of the current directory build from the latest Lucene commit.
      * Only files that are part of the last commit are considered in this datastrucutre.
-     * For backwards compatibility the snapshot might include legacy checksums that
+     * For backwards compatibility, the snapshot might include legacy checksums that
```
Comment (nik9000) [inline core/src/main/java/org/elasticsearch/index/store/Store.java]: This `,` is optional in English. I don't tend to include it most of the time so I wouldn't change this.

**Comment by nikiforosbotis** [comment]
@nik9000 I did the corrections according to your comments. For anything else that may be needed, please let me know. Thank you.

**Comment by nik9000** [comment]
Thanks for doing those @nikiforosbotis! I've merged and backported to 5.x and 5.3.

**Comment by nikiforosbotis** [comment]
Thank you @nik9000 ! I will try to do more improvements at the elastic search's source code.

**Pull request merged**